### PR TITLE
Regenerate Monitoring and author synth script

### PR DESCRIPTION
**Pull request by crwilcox**

### CODE REVIEW

File: monitoring/synth.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICGenerator()
+common = gcp.CommonTemplates()
+
+# tasks has two product names, and a poorly named artman yaml
```
Comment (theacodes) [inline monitoring/synth.py]: seems to be copy-pasta? Says `tasks` and using `v2beta2_library`, but this is monitoring v3.

**Comment by crwilcox** [comment]
Docs build is failing and will need a fix.
```
Warning, treated as error:
/var/code/gcp/.nox/docs/lib/python3.6/site-packages/google/cloud/monitoring_v3/__init__.py:docstring of google.cloud.monitoring_v3.AlertPolicyServiceClient.update_alert_policy:39:Bullet list ends without a blank line; unexpected unindent.
```

**Comment by theacodes** [comment]
You'll need to fix that in the synth script. :(

**Comment by crwilcox** [comment]
Yep. just wanted to note here what it was. Fix incoming.

**Pull request merged**